webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateComponents/TodayButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../../../common/I18n';
+
+class TodayButton extends React.Component {
```

Comment:
no state nor lifecycle methods, can you make it as functional component?

Reply:
Changed, thanks